<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerConnectionSingleLineWidget</class>
 <widget class="QWidget" name="ServerConnectionSingleLineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>50</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leIp">
       <property name="inputMask">
        <string extracomment="ip">000.000.000.000;_</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="placeholderText">
        <string>set ip host address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPort">
       <property name="specialValueText">
        <string/>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>25500</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tblAddConnection">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../zaviruha_r/src/view/app/images/ActionsListAdd.png</normaloff>../../../../../../zaviruha_r/src/view/app/images/ActionsListAdd.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tblRemoveConnection">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../app/rc.qrc">
         <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../app/rc.qrc"/>
 </resources>
 <connections/>
</ui>
